src/azul/transformer.py

```diff
@@ -167,7 +176,12 @@ def translate_fields(cls,
                         new_dict[key + '_'] = val
             return new_dict
         elif isinstance(doc, list):
-            return [cls.translate_fields(val, field_types, forward=forward) for val in doc]
+            # Translate [] to [null_string] but only at the field level (to avoid case of contents['organoids'] == [])
```

Comment:
```suggestion
            # Translate an empty list to a list containing a single None value (and then further translate that None value according to the field type), but do so only at the field level (to avoid case of contents['organoids'] == []).
```

And wrap at 80